package.json

```diff
@@ -39,7 +39,7 @@
     "express-passport-logout": "~0.1.0",
     "express-session": "^1.15.6",
     "freeport": "~1.0.2",
-    "jsonwebtoken": "^5.7.0",
+    "jsonwebtoken": "^8.5.1",
```

Comment:
What is the plan to test any impact due to major upgrade?

Reply:
1. review changes introduced between 5.7.0 and 8.5.1
2. fix breaking changes
3. test affected scenarios in Vivaldi (OpenLDAP) and Windows Server (AD LDAP)